cmd/otel-allocator/server/server.go

```diff
@@ -107,7 +107,12 @@ func (s *Server) Shutdown(ctx context.Context) error {
 	return s.server.Shutdown(ctx)
 }
 
-func RemoveRelabelAction(jsonConfig []byte) ([]byte, error) {
+// This is needed specifically for keepequal/dropequal actions because even though the user doesn't specify the
```

Comment:
```suggestion
// RemoveRegexFromRelabelAction is needed specifically for keepequal/dropequal actions because even though the user doesn't specify the
```